decompiler/backend/cexpressiongenerator.py

```diff
@@ -180,6 +181,10 @@ def visit_list_operation(self, op: operations.ListOperation) -> str:
 
     def visit_unary_operation(self, op: operations.UnaryOperation) -> str:
         """Return a string representation of the given unary operation (e.g. !a or &a)."""
+        if isinstance(op, MemberAccess):
+            if isinstance(op.struct_variable.type, Pointer):
+                return f"{self.visit(op.struct_variable)}->{op.member_name}"
+            return f"{self.visit(op.struct_variable)}.{op.member_name}"
```

Comment:
I could do that,  but it would be just `.`

`MemberAccess` is a class for both struct member access by pointer (`->` syntax) and direct (`.` member access).
Default is the latter though.

Reply:
Done.